### PR TITLE
CNV20024: Add MigrationPolicies list empty state

### PR DESCRIPTION
**Pull request by avivtur**
Signed-off-by: Aviv Turgeman <aturgema@redhat.com>
## 📝 Description

Adding empty state to MigrationPolicies list page

## 🎥 Demo

Screenshot (90)

### CODE REVIEW

File: src/views/migrationpolicies/list/components/MigrationPoliciesEmptyState/MigrationPoliciesEmptyState.tsx [hunk 1]
```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import { Trans } from 'react-i18next';
+import { useHistory } from 'react-router-dom';
+import MigrationPolicyEmptyListImage from 'images/migrationPoliciesEmptyState.png';
```
Comment (metalice) [inline src/views/migrationpolicies/list/components/MigrationPoliciesEmptyState/MigrationPoliciesEmptyState.tsx:4]:
```suggestion
import {ReactComponent as MigrationPolicyEmptyListImage} from 'images/migrationPoliciesEmptyState.png';
```

can try to see if its working for u? its working for svgs, not sure about pngs

File: src/views/migrationpolicies/list/components/MigrationPoliciesEmptyState/MigrationPoliciesEmptyState.tsx [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+          Click <b>Create MigrationPolicy</b> to create you first policy
+        </Trans>
+      </EmptyStateBody>
+      <EmptyStatePrimary>
+        <Button
+          variant={ButtonVariant.primary}
+          onClick={() => history.push(`${migrationPoliciesPageBaseURL}/form`)}
+        >
+          {t('Create MigrationPolicy')}
+        </Button>
```
Comment (metalice) [inline src/views/migrationpolicies/list/components/MigrationPoliciesEmptyState/MigrationPoliciesEmptyState.tsx:42]: you can use a link here as button and avoid using history push

File: src/views/migrationpolicies/list/components/MigrationPoliciesEmptyState/MigrationPoliciesEmptyState.tsx (hunk 1 above)
Comment (avivtur) [reply src/views/migrationpolicies/list/components/MigrationPoliciesEmptyState/MigrationPoliciesEmptyState.tsx:4]: tried to do it and it doesn't work for me

File: src/views/migrationpolicies/list/components/MigrationPoliciesEmptyState/MigrationPoliciesEmptyState.tsx (hunk 2 above)
Comment (avivtur) [reply src/views/migrationpolicies/list/components/MigrationPoliciesEmptyState/MigrationPoliciesEmptyState.tsx:42]: I need the `ButtonVariant.primary` CSS styles to the button, you suggest switching to Link and adding those styles?

**Pull request merged**